applications/admin.py

```diff
@@ -93,13 +95,38 @@ class Meta:
 
 @admin.register(ApplicationRound)
 class ApplicationRoundAdmin(admin.ModelAdmin):
+    actions = ["export_to_csv"]
     form = ApplicationRoundAdminForm
     model = ApplicationRound
     inlines = [ApplicationRoundStatusInline, ApplicationRoundBasketInline]
     autocomplete_fields = [
         "reservation_units",
     ]
 
+    @admin.action(description="Export first selected reservation unit to CSV")
+    def export_to_csv(self, request, queryset):
+        try:
+            app_round = queryset.first()
+            path = ApplicationDataExporter.export_application_data(
+                application_round=app_round
+            )
+
+        except Exception as e:
+            self.message_user(
```

Comment:
Should we log these to sentry so that we are not depending on user's screenshot when debuging possible errors? Simple `capture_exception(e)` should do the trick.

Reply:
very good idea, i'll do that